libs/net/dal/Migrations/1.3.8/Up/PostUp/00-source-series.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+
+ALTER TABLE public."source" ADD COLUMN "is_cbra_source" boolean NOT NULL DEFAULT false;
```

Comment:
Why do you need these?  EF migration should handle all of this for you.

Reply:
so I don't need this PostUp at all?

Reply:
updated. thanks